citesphere-importer/src/test/java/edu/asu/diging/citesphere/importer/core/service/parse/crossref/CrossRefParserTest.java

```diff
@@ -0,0 +1,213 @@
+package edu.asu.diging.citesphere.importer.core.service.parse.crossref;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import edu.asu.diging.citesphere.importer.core.model.impl.ArticleMeta;
+import edu.asu.diging.citesphere.importer.core.model.impl.ContainerMeta;
+import edu.asu.diging.citesphere.importer.core.model.impl.Issn;
+import edu.asu.diging.citesphere.importer.core.service.parse.impl.CrossRefParser;
+import edu.asu.diging.crossref.model.Date;
+import edu.asu.diging.crossref.model.IssnType;
+import edu.asu.diging.crossref.model.Item;
+import edu.asu.diging.crossref.model.Person;
+import edu.asu.diging.crossref.model.Review;
+import edu.asu.diging.crossref.model.impl.IssnTypeImpl;
+
+public class CrossRefParserTest {
+    private CrossRefParser crossRefParser;
+
+    @Before
+    public void setUp() {
+        crossRefParser = new CrossRefParser();
+    }
+
+    @Test
+    public void testParseJournalMeta_WithValidData() {
```

Comment:
test method names should follow our naming convention (documented in confluence)